File: BackEnd/mimi/src/main/java/com/mimi/Dto/Store.java
package com.mimi.Dto;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
@ToString
@Getter
@Setter
@Document(collection="store")
public class Store {
	@Id
	private int id;
	private int bID;
	private String name;
	private String address;
	private String tel;
	private String category;
	private String mainMn;
	private String price;
	private String menu;
	private Double rating;
	private int rvwCnt;
	private String img;
	private String tags;

	
}

File: App/app/src/main/java/com/chd/mimitogether/ui/party/dto/Store.kt
package com.chd.mimitogether.ui.party.dto

import java.io.Serializable

data class Store(
    val id: Int,
    val boID: String,
    val name: String,
    val address: String,
    val tel: String,
    val category: String,
    val mainMn: String,
    val price: String,
    val menu: String,
    val rating: Double,
    val rvwCnt: Int,
    val img: String,
    val tags: String
): Serializable